editor/src/clj/editor/ui.clj

```diff
@@ -1500,7 +1500,7 @@
   (when-some [parent (or (.getParentMenu old) menu-bar)]
     (when-some [parent-children (menu-items parent)]
       (let [index (.indexOf parent-children old)]
-        (when (pos? index)
+        (when-not (neg? index)
```

Comment:
Bugfix: `(ui/invalidate-menubar-item! ::file)` wouldn't do anything since it is at index zero.